src/main/java/uk/gov/hmcts/reform/preapi/services/RecordingService.java

```diff
@@ -26,13 +29,19 @@ public class RecordingService {
     private final RecordingRepository recordingRepository;
     private final BookingRepository bookingRepository;
     private final CaptureSessionRepository captureSessionRepository;
+    private final ShareRecordingRepository shareRecordingRepository;
```

Comment:
You are going to need some mock beans in the tests for the service (file called `RecordingServiceTest`) 
```java
    @MockBean
    private ShareRecordingRepository shareRecordingRepository;

    @MockBean
    private UserRepository userRepository;
```